### ISSUE TITLE
Distinguish between merge requests by package manager at a glance

### ISSUE THREAD

**Issue by cschroed-usgs** [renovatebot/renovate#18272]
### What would you like Renovate to be able to do?

# Scenario
A cross functional team containing frontend and backend developers are contributing to a polyglot monorepo whose backend is in Python and whose frontend is in JavaScript.

# User Stories:
As a frontend developer working in a polyglot monorepo, I want to understand at a glance which renovate-generated merge requests relate to npm dependencies so that I can skip merge requests related to pip, and instead focus on the merge requests I am best-qualified to review.

The mirror of the user story...

As a backend developer working in a polyglot monorepo, I want to understand at a glance which renovate-generated merge requests relate to pip dependencies so that I can skip merge requests related to npm, and instead focus on the merge requests I am best-qualified to review.

### If you have any ideas on how this should be implemented, please tell us here.

A few alternatives...

1. Add labels to merge requests that correspond to the package manager. If a merge request updates an npm dependency, it would have label "npm". This would allow users of GitLab or GitHub to filter the list of open merge requests by setting `label = "npm"`.
2. Embed the name of the package manager into the title at a predictable place. This would allow users of GitLab or Github to filter the list of open merge requests by title using fuzzy matching. `title ~"npm"`

### Is this a feature you are interested in implementing yourself?

Maybe

**Comment by viceice** [comment]
Labels should already work.

you should also be able to add additional information to the PR title.

**Comment by cschroed-usgs** [comment]
Thanks for taking a look at this @viceice ! Where do you see this working now? No labels specific to package managers are currently automatically added to the merge requests renovate generates on our self-hosted GitLab instance. Does Renovate this functionality available on other source control solutions?

**Comment by viceice** [comment]
no, you need to configure package rules for it. renovate won't do it out of the box.

**Comment by HonkingGoose** [comment]
This older issue of mine is related:

- #7485

---

Example of a `packageRule` to get labels for `npm` and Python packages:

```json
{
  "packageRules": [
    {
      "matchManagers": ["npm"],
      "labels": ["npm"]
    },
    {
      "matchManagers": ["pip-compile", "pip_requirements", "pip_setup", "pipenv", "poetry", "pyenv", "setup-cfg"]
      "labels": ["pip"]
    }
  ]
}
```
